pxtlib/main.ts

```diff
@@ -230,6 +230,11 @@ namespace pxt {
                     appTheme: {
                         hideMenuBar: true
                     }
+                },
+                "sidebarTutorial=1": {
```

Comment:
why did you decide to define this here instead of the pxtarget.json in pxt-minecraft?

Reply:
oo I didn't even consider putting it in pxt-minecraft. I can do that!